build.sbt

```diff
@@ -17,8 +17,8 @@ lazy val projectSettings = Seq(
   version := "0.1.0-SNAPSHOT",
   resolvers ++=
     Resolver.sonatypeOssRepos("releases") ++
-    Resolver.sonatypeOssRepos("snapshots") ++
-    Seq("jitpack" at "https://jitpack.io"),
+      Resolver.sonatypeOssRepos("snapshots") ++
+      Seq("jitpack" at "https://jitpack.io"),
```

Comment:
Is jitpack used for something else anymore?

Reply:
Kalium library is there, we use it for Ed25519. Which we do not use now. So maybe it makes sense to remove both.